### ISSUE TITLE
ItemJoin failed thread check for reason: Asynchronous getNearbyEntities

### ISSUE THREAD

**Issue by mibby** [RockinChaos/ItemJoin#335]
ItemJoin dev 570
Tuinity dev 399 (Paper Fork - Spigot 1.16.3)
https://github.com/Spottedleaf/Tuinity/actions

```
[23:52:51] [Craft Scheduler Thread - 170 - ItemJoin/FATAL]: Thread Craft Scheduler Thread - 170 - ItemJoin failed thread check for reason: Asynchronous getNearbyEntities
java.lang.Throwable: null
	at org.spigotmc.AsyncCatcher.catchOp(AsyncCatcher.java:15) ~[patched_1.16.3.jar:git-Tuinity-"f7b8e25"]
	at org.bukkit.craftbukkit.v1_16_R2.entity.CraftEntity.getNearbyEntities(CraftEntity.java:581) ~[patched_1.16.3.jar:git-Tuinity-"f7b8e25"]
	at me.RockinChaos.itemjoin.handlers.PlayerHandler.getNearbyPlayer(PlayerHandler.java:414) ~[?:?]
	at me.RockinChaos.itemjoin.utils.Utils.translateLayout(Utils.java:566) ~[?:?]
	at me.RockinChaos.itemjoin.item.ItemMap.setCustomName(ItemMap.java:3461) ~[?:?]
	at me.RockinChaos.itemjoin.item.ItemMap.updateItem(ItemMap.java:3203) ~[?:?]
	at me.RockinChaos.itemjoin.item.ItemMap.removeFrom(ItemMap.java:3668) ~[?:?]
	at me.RockinChaos.itemjoin.item.ItemUtilities.lambda$handleItems$3(ItemUtilities.java:229) ~[?:?]
	at me.RockinChaos.itemjoin.handlers.ServerHandler.lambda$runAsyncThread$0(ServerHandler.java:43) ~[?:?]
	at org.bukkit.craftbukkit.v1_16_R2.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.3.jar:git-Tuinity-"f7b8e25"]
	at org.bukkit.craftbukkit.v1_16_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:58) ~[patched_1.16.3.jar:git-Tuinity-"f7b8e25"]
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.16.3.jar:git-Tuinity-"f7b8e25"]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) [?:?]
	at java.lang.Thread.run(Thread.java:832) [?:?]
```

https://paste.gg/p/anonymous/8254457e227641d6a33cfabde6b6b7a9/files/28b89dbc622042aea5a630a27d0dc102/raw

Config
https://paste.gg/p/anonymous/f955004d595c4f7f8810a2519d698fd9/files/3925d2ebf6ff4b48b29ae89e5106c186/raw

Items
https://paste.gg/p/anonymous/5bdcfeab66044ffca504a86ed6dee0cd/files/f5ba49471c0c427f9b5f05b5468966b5/raw

**Comment by RockinChaos** [comment]
The latest snapshot should now fix this issue; https://ci.craftationgaming.com/job/ItemJoin/576/

Let me know!

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.2.0.
You can grab it here; Download: https://www.spigotmc.org/resources/itemjoin.12661/download?version=354306

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes
